views/unsubscribe.hbs

```diff
@@ -1,48 +1,49 @@
-          <img class="loader" src="/img/loader.gif" alt="loading data" />
-        </div>
-      </form>
-    </div>
-  </section>
-</div>
-    {{ else }}
-    <h2 class="section-headline whole">Unsubscribe from Firefox Monitor</h2>
-    <p class="whole sub-head">
-      This will remove your email from the Firefox Monitor list and you will no longer receive alerts when new breaches are announced.
-    </p>
-    <div class="half">
-      <form action="/user/unsubscribe" class="form-group" method="post" id="unsubscribe-form">
-        <input type="hidden" name="token" value="{{ token }}">
-        <input type="hidden" name="emailHash" value="{{ hash }}">
-        <div class="input-group-button">
-          <input type="submit" class="button submit transparent-button" value="Unsubscribe">
-          <img class="loader" src="/img/loader.gif" alt="loading data" />
-        </div>
-      </form>
-    </div>
+    </section><!--.unsubscribe-content.section-wrapper-->
+    <section class="unsubscribe-survey section-wrapper drop-shadow">
+      <span class="secondary-title whole">Why are you unsubscribing from Firefox Monitor alerts?</span>
+      <div class="whole">
+        <form action="" method="post" id="unsubscribe-survey-form" class="form-group">
+          {{#each UNSUB_REASONS}}
+            <div class="radio-button-group">
+              <input id="reason{{ id_number }}" type="radio" name="unsubscribe-reason" value="{{ value }}" tabindex="1" />
+              <span class="radio-dot"></span>
+              <label for="reason{{ id_number }}" tabindex="1">{{ value }}</label>
+            </div>
+          {{/each}}
+          <div class="input-group-button">
+            <input id="unsubscribe" type="submit" class="button submit transparent-button" value="Unsubscribe" tabindex="0"/>
+            <img class="loader" src="/img/loader.gif" alt="loading data" />
+            <span id="invalid-unsubscribe-reason" class="error-message">Please choose an answer.</span>
+          </div>
+        </form>
+      </div>
+    </section>
+  {{ else }}
+    <section class="unsubscribe-content section-wrapper">
+      <h2 class="section-headline whole">Unsubscribe from Firefox Monitor</h2>
+      <p class="whole sub-head">
+        This will remove your email from the Firefox Monitor list and you will no longer receive alerts when new breaches are announced.
+      </p>
+      <div class="half">
+        <form action="/user/unsubscribe" class="form-group" method="post" id="unsubscribe-form">
+          <input type="hidden" name="token" value="{{ token }}">
+          <input type="hidden" name="emailHash" value="{{ hash }}">
+          <div class="input-group-button">
+            <input type="submit" class="button submit transparent-button" value="Unsubscribe">
+            <img class="loader" src="/img/loader.gif" alt="loading data" />
+          </div>
+        </form>
+      </div>
+    {{/if}}
```

Comment:
My brain hurts. I _want_ to say that the `</section>` closing tag should be within the `{{/if}}`, but I think I'd need to copy/pasta this into Sublime and try cutting the `if..else` branches individually to make sure each branch has the correct tag nesting.